app.js

```diff
@@ -58,24 +60,20 @@ app.post('/', async (req, res, next) => {
         memberImagebuffers.push(membersJSON[memberName]);
         handshakeRanks.push(`${limitText(memberName)} - ${handshakes[i].sum} kali` || null);
       }
-    }
-
-    if (handshakes.length === 0) {
+    } else {
```

Comment:
else will prone to error if we have unhandled case within the ifs, in this case, we know every possible case, so it's okay.